README.md

```diff
@@ -1,46 +1,163 @@
 # Catalyst
 
+The entire bundle of application needed for booting up livepeer's
+decentralised media server in place is termed as [catalyst].
+
 ## Getting Started
 
-go1.17 is recommended because building with < 1.17 (specifically 1.16) can result in issue on arm64 architectures.
+Make sure you have the following installed on the machine before
+proceeding:
 
-From there:
+  - go1.17± or newer
+  - ffmpeg v5 (or newer)
+  - GNU make (not required, but good to have for using `Makefile`
+    commands)
+  - docker version 19 (or newer) (not mandatory for basic catalyst
+    operations)
+  - cloning this repository alongside [go-livepeer], [mistserver] and
+    other dependent projects of [catalyst] in the same directory
+  - about 2-3 GB of free disk space
 
-```
-# boots up Postgres and RabbitMQ dependencies
-make docker-compose
+±: go1.17 is recommended because building with < 1.17 (specifically 1.16)
```

Comment:
I think we need 1.19 because of upcoming [url.JoinPath()](https://github.com/livepeer/catalyst-api/blob/transcode/handlers/misttriggers/live_tracklist.go#L84)